stage1/common/run.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stage1_common
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/rkt/pkg/sys"
+)
+
+func WithClearedCloExec(lfd int, f func() error) error {
+	err := sys.CloseOnExec(lfd, false)
+	if err != nil {
+		return err
+	}
+	defer sys.CloseOnExec(lfd, true)
+
+	return f()
+}
+
+func WritePpid(pid int) error {
+	// write ppid file as specified in
```

Comment:
now that this is an exported function can you promote this comment to being a docstring?